src/libs/deploy.js

```diff
@@ -168,15 +169,57 @@ export const deployGoogle = async (signedApp, workspace = '/tmp/') => {
 };
 
 /**
- * Apple Store Deployment
+ * App Store Deployment
  *
- * @param {String} signedApp The name of the signed app
+ * @param {String} signedApp The name of the signed app, ONLY accepts .ipa / .pkg format
  * @param {string} [workspace='/tmp/'] The workspace to use
  * @returns The status of the deployment
  */
-// eslint-disable-next-line no-unused-vars
-export const deployAppleStore = async (signedApp, workspace = '/tmp/') => {
-  // TODO
+// eslint-disable-next-line import/prefer-default-export
+export const deployAppStore = async (signedApp, workspace = '/tmp/') => {
+  // Get app:
+  const signedAppPath = await fetchFileFromStorage(signedApp, workspace);
```

Comment:
I fee like I've reviewed this code already. In any case, `await` needs to be wrapped in a try / catch else you'll probably get unhandled promise rejection warnings.